### PR TITLE
adding option to provide context to ambari operations

### PR DESCRIPTION
**Pull request by jhsenjaliya**
having context on ambari operation is useful to distinguish between actor on ambari.

### CODE REVIEW

File: ambariclient/models.py [hunk 1]
```diff
@@ -238,13 +238,16 @@ def _server_components(self):
 
         return server_components
 
-    def install(self):
+    def install(self, context):
```
Comment (jimbobhickville) [inline ambariclient/models.py]: You should make these not required by either doing:
```context=None```
or just putting the default string in the parameter list:
```context="Install All Host Components"```

File: ambariclient/models.py [hunk 2]
```diff
@@ -896,7 +917,8 @@ def recommission(self, service, hosts):
             for host in hosts:
                 self.hosts(host).components("NODEMANAGER").start()
 
-    def commission(self, service, hosts, commission_type):
+
+def commission(self, service, hosts, commission_type):
```
Comment (jimbobhickville) [inline ambariclient/models.py]: I think this de-indentation was accidental and will break things.

File: ambariclient/models.py (hunk 1 above)
Comment (jhsenjaliya) [inline ambariclient/models.py]: yes, will do that

**Comment by jhsenjaliya** [comment]
@jimbobhickville, I have updated this PR as per your review. Please check when you get chance.

**Comment by jhsenjaliya** [comment]
@jimbobhickville Thanks !

**Pull request merged**